src/cmake/modules/FindOpenColorIO.cmake

```diff
@@ -24,13 +24,15 @@ include (FindPackageMessage)
             message(STATUS "OCIO LIBRARY_PATH explicitly specified: ${OCIO_LIBRARY_PATH}")
         endif()
     endif ()
-    FIND_PATH( OCIO_INCLUDES OpenColorIO/OpenColorIO.h
+    FIND_PATH(OCIO_INCLUDES
+        OpenColorIO.h
         ${OCIO_INCLUDE_PATH}
```

Comment:
I'm not sure this is correct. The shorthand is

    FIND_PATH (OCIO_INCLUDES name path path path)

But if you use the longhand version, such as with PATH_SUFFIXES, you seem to also need the token PATHS here.

At least, it fails for me if I don't and works for me if I do, and that appears to be what is described by the docs here: https://cmake.org/cmake/help/v3.0/command/find_path.html